database/pSQLdatabase.js

```diff
@@ -0,0 +1,80 @@
+/* eslint-disable no-console */
+/* eslint-disable max-classes-per-file */
+const { Sequelize, Model, DataTypes } = require('sequelize');
+const dotenv = require('dotenv');
+
+dotenv.config();
+const sequelize = new Sequelize('postgres://127.0.0.1:5432/SDC');
+
+class Price extends Model { }
+Price.init({
+  courseId: {
+    type: DataTypes.INTEGER,
+    unique: true,
+    primaryKey: true,
+  },
+  basePrice: DataTypes.DECIMAL,
+  discountPercentage: DataTypes.INTEGER,
+  saleEndDate: DataTypes.DATE,
+  saleOngoing: {
+    type: DataTypes.BOOLEAN,
+    defaultValue: false,
+  },
+}, { sequelize });
+
+class PreviewVideo extends Model { }
+PreviewVideo.init({
+  courseId: {
+    type: DataTypes.INTEGER,
+    unique: true,
+    primaryKey: true,
+  },
+  previewVideoUrl: DataTypes.STRING,
+  previewVideoImgUrl: DataTypes.STRING,
+}, { sequelize });
+
+class Sidebar extends Model { }
+Sidebar.init({
+  courseId: {
+    type: DataTypes.INTEGER,
+    unique: true,
+    primaryKey: true,
+  },
+  fullLifetimeAccess: DataTypes.STRING,
+  accessTypes: DataTypes.STRING,
+  assignments: {
+    type: DataTypes.BOOLEAN,
+    defaultValue: false,
+  },
+  certificateOfCompletion: {
+    type: DataTypes.BOOLEAN,
+    defaultValue: true,
+  },
+  downloadableResources: DataTypes.INTEGER,
+}, { sequelize });
+
+const openConn = async () => {
+  try {
+    await sequelize.authenticate();
+    console.log('DB connection successful.');
+    await Price.sync({ force: true });
```

Comment:
what was the reason for including force: true initially?

Reply:
When I was trying to usee sequelize to populate my DB i wanted it to drop whatever was existing and populate on a "fresh" db.